backend/open_api_spec.yml

```diff
@@ -0,0 +1,790 @@
+            application/json:
+              schema:
+                type: object
+                properties:
+                  access_token:
+                    type: string
+        "401":
+          $ref: "#/components/responses/401"
+
+  /curation/collections:
+    get:
+      description: Fetch all Collections metadata. Authentication required for private collections.
+      security:
+        - cxguserCookieLenient: []
+      parameters:
+        - $ref: "#/components/parameters/query_visibility"
+      responses:
+        "200":
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  collections:
+                    type: array
+                      items:
+                        type: object
+                          required:
+                            - id
+                            - name
+                            - visibility
+                            - contact_name
+                            - contact_email
+                            - curator_name
+                            - revised_at
+                            - created_at
+                            - links
+                            - datasets
+                          properties:
+                            id:
+                              $ref: "#/components/schemas/collection_uuid"
+                            name:
+                              type: string
+                            visibility:
+                              $ref: "#/components/schemas/visibility"
+                            contact_name:
+                              type: string
+                            contact_email:
+                              type: string
+                            curator_name:
+                              type: string
+                            revised_at:
+                              type: number
+                            created_at:
+                              type: number
+                            links:
+                              $ref: "#/components/schemas/links"
+                            datasets:
+                              type: array
+                              items:
+                                $ref: "#/components/schemas/dataset_preview"
+                            revision_of:
+                              type: string
+                              description: uuid of the public version of this Collection
+                            access_type:
+                              type: string
+                              enum: [ READ, WRITE ]
+                            published_at:
+                              type: number
+                            description:
+                              type: string
+                            obfuscated_uuid:
+                              type: string
+                            data_submission_policy_version:
```

Comment:
`data_submission_policy_version` is not particularly useful for curators. It's only set when the collection is published or revised/published when the curator accepts the data submission policy on behalf of the data submitter. 

It's only updated when the policy is updated - which has happened only once. And when it's updated, there's an out-of-band method for getting permission from all the curators to update the state in legacy collections via a script.